src/style/site.css

```diff
@@ -576,7 +576,7 @@ ul.footer-icons li {
 .notes-place-holder {
   height: 548px;
   padding-top:175px;
-  background-color: #ecf0f1;;
+  background-color: #ecf0f1;
```

Comment:
The `well` class applies this `background-color`. This field shouldn't be needed.